src/main/java/signup_and_login/UserModel.java

```diff
@@ -0,0 +1,29 @@
+package signup_and_login;
+
+public class UserModel {
```

Comment:
I think UserModel object is used for encapsulating inputs from users and encapsulating data from the database. How about dividing it into two classes to follow SRP?

Reply:
see commit 26deba1d40f81820e9a3b6ff5cc79ea906e3af91